seatunnel-transforms-v2/src/main/java/org/apache/seatunnel/transform/nlpmodel/ModelProvider.java

```diff
@@ -26,6 +26,7 @@ public enum ModelProvider {
             "https://ark.cn-beijing.volces.com/api/v3/embeddings"),
     QIANFAN("", "https://aip.baidubce.com/rpc/2.0/ai_custom/v1/wenxinworkshop/embeddings"),
     KIMIAI("https://api.moonshot.cn/v1/chat/completions", ""),
+    MICROSOFT("", ""),
```

Comment:
So when use microsoft model, user must configure api path? we should let users know this.

Reply:
get，I changes